pkg/ebpf/ingress_node_firewall_loader.go

```diff
@@ -220,6 +220,12 @@ func (infc *IngNodeFwController) IngressNodeFwAttach(ifacesName []string, isDele
 				Interface: iface.Index,
 			})
 			if err != nil {
+				if errno, ok := err.(syscall.Errno); ok {
+					if errno == syscall.EBUSY {
```

Comment:
That doesn't work because cilium return a string type error:
~~~
2022/08/09 10:41:34 akaris ---------------- can't create link: device or resource busy *errors.errorString
~~~

Here /development/ingress-node-firewall/vendor/github.com/cilium/ebpf/link/link.go:
~~~
  178     fd, err := sys.LinkCreate(&attr)                                                                                    
  179     if err != nil {                                                                                                     
  180         return nil, fmt.Errorf("can't create link: %s", err)                                                            
  181     } 
~~~

Reply:
This here works though:
~~~
  223             if err != nil {                                                                                             
  224                 if strings.Contains(err.Error(), "device or resource busy") {                                           
  225                     log.Printf("Interface %s is already attached", ifaceName)                                           
  226                     return ifMap, nil                                                                                   
  227                 }                                                                                                       
  228                 return ifMap, fmt.Errorf("could not attach XDP program: %s", err)                                       
  229             }  
~~~

Reply:
yep that is what I did :)